vyper/__init__.py

```diff
@@ -12,7 +17,25 @@
     raise Exception("Requires python3.6+")  # pragma: no cover
 
 
+def _get_version_hash(root: _Path) -> str:
+    m = _hashlib.sha1()
+    for dirname, _, file_list in _os.walk(root):
+        for fname in file_list:
+            if fname[-3:] == '.py':
+                with open(_Path(dirname).joinpath(fname), 'rb') as f:
+                    m.update(f.read())
+    return m.digest().hex()
```

Comment:
@fubuloubu couldn't we just subprocess this to git instead? `git rev-parse --short HEAD` I am worried this causes more confusion.

Reply:
Does snapcraft, docker, and PyPI all have git?